src/layer/vector/Renderer.js

```diff
@@ -113,16 +54,15 @@ export const Renderer = Layer.extend({
 		}
 	},
 
-	_update() {
-		// Update pixel bounds of renderer container (for positioning/sizing/clipping later)
-		// Subclasses are responsible of firing the 'update' event.
-		const p = this.options.padding,
-		    size = this._map.getSize(),
-		    min = this._map.containerPointToLayerPoint(size.multiplyBy(-p)).round();
+	_onViewReset() {
+		for (const id in this._layers) {
+			this._layers[id]._reset();
+		}
+	},
 
-		this._bounds = new Bounds(min, min.add(size.multiplyBy(1 + p * 2)).round());
+	_onSettled() {
+		// Subclasses are responsible of firing the 'update' event.
+		this._update();
```

Comment:
The function `_update()` doesn't exist in Renderer, maybe you want to add a false-function?

Reply:
The comment from https://github.com/Leaflet/Leaflet/pull/8611/commits/931d98c974c0d0a7a39d772c50ee04552655f0c0 should clarify this. Basically I want to maintain the code from L.Canvas and L.SVG unchanged, and at the same time `_update` feels ambiguous.

Reply:
Some plugins are extending from `Renderer` so we should have no unimplemented methods, which are called. Currently it is more hacky then a clean solution with `_update()`.
We should just add `_update: Util.falseFn,`

Reply:
You have a good point. I've added that.